src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectEditingPreferencesPane.java

```diff
@@ -37,44 +34,54 @@
 import org.rstudio.studio.client.workbench.views.source.editors.text.ui.ChooseEncodingDialog;
 import org.rstudio.studio.client.workbench.views.source.model.SourceServerOperations;
 
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.Style.Unit;
 import com.google.gwt.event.dom.client.ClickEvent;
 import com.google.gwt.event.dom.client.ClickHandler;
 import com.google.gwt.resources.client.ImageResource;
 import com.google.gwt.user.client.ui.CheckBox;
+import com.google.gwt.user.client.ui.Label;
 import com.google.inject.Inject;
 
 public class ProjectEditingPreferencesPane extends ProjectPreferencesPane
 {
    @Inject
    public ProjectEditingPreferencesPane(final SourceServerOperations server)
    {
+      add(headerLabel("Editing"));
```

Comment:
Suggestion to add constants and the _en/_fr entries for any of the `add(headerLabel("text"))` text